src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -435,6 +447,7 @@ impl AccessChecker for PrivilegeAccess {
             .get_enable_experimental_rbac_check()?;
         let tenant = self.ctx.get_tenant();
         let catalog_name = self.ctx.get_current_catalog();
+        let if_exists = false;
```

Comment:
how about removing this variable, and just pass `false` in these call sites?

I guess in most cases we do not really need to care about whether `if_exists` or not, but only a few cases like `RenameTable` `DropTable`, we need only care these few cases to get the `if_exists` from the plan.

Reply:
Ok, if add new if exists, just need replace false